shapeshift/web
# add configuration management

## Notice

Before submitting a pull request, please make sure you have answered the following:


## Pull Request Type


## Issue (if applicable)

closes #48 #29 

## Description

Adds a centralized place to manage config. Validates and strongly types config.

## Testing

1. run the app without adding `REACT_APP_UNCHAINED_URL` to your `.env`
2. it should be really angry with you
3. add it to your config
4. it won't be angry any more

## Screenshots (if applicable)
